copyspecial.py

```diff
@@ -10,7 +10,8 @@
 import re
 import os
 import shutil
-import commands
+# import commands
+import subprocess
```

Comment:
Nice, the commands module was deprecated in Python 2.6 in favor of the subprocess module.